service/matching/matching_engine_test.go

```diff
@@ -2964,18 +3356,45 @@ func (m *testTaskManager) GetTaskQueue(
 	}
 	return &persistence.GetTaskQueueResponse{
 		TaskQueueInfo: &persistencespb.TaskQueueInfo{
-			NamespaceId:    request.NamespaceID,
-			Name:           request.TaskQueue,
-			TaskType:       request.TaskType,
-			Kind:           enumspb.TASK_QUEUE_KIND_NORMAL,
-			AckLevel:       tlm.ackLevel,
-			ExpiryTime:     nil,
-			LastUpdateTime: timestamp.TimeNowPtrUtc(),
+			NamespaceId:             request.NamespaceID,
+			Name:                    request.TaskQueue,
+			TaskType:                request.TaskType,
+			Kind:                    enumspb.TASK_QUEUE_KIND_NORMAL,
+			AckLevel:                tlm.ackLevel,
+			ExpiryTime:              nil,
+			LastUpdateTime:          timestamp.TimeNowPtrUtc(),
+			ApproximateBacklogCount: tlm.ApproximateBacklogCount,
 		},
 		RangeID: tlm.rangeID,
 	}, nil
 }
 
+// minTaskID is a helper to return the minimum value of the TaskID present in testTaskManager
+func (m *testTaskManager) minTaskID(dbq *PhysicalTaskQueueKey) (int64, bool) {
+	tlm := m.getQueueManager(dbq)
+	tlm.Lock()
+	defer tlm.Unlock()
+	minKey, _ := tlm.tasks.Min()
+	key, ok := minKey.(int64)
+	if ok == false {
```

Comment:
```suggestion
	if !ok {
```

Reply:
but you can also just `return key, ok`. the caller will ignore key if ok is false, so it doesn't matter if it's zero, right?

Reply:
I agree - shall change this to return `key, ok`.